### DISCUSSION TITLE
RANGO DE ID’S EN LOS INPUTS

### DISCUSSION THREAD

**Discussion by Consolo0** [IIC2133-PUC/2024-1#27] [Tarea 0] [answered]
Cuando nos entregan ID's de las naves o de los planetas para las distintas funciones, podemos asumir que estos son válidos (están dentro del rango dado en el input al principio del código) o es nuestro deber chequear eso en cada función.

**Comment by benjavicente** [comment, marked as answer]
Se puede asumir que son válidos.
